### ISSUE TITLE
Spot checks for Scenery input changes

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/QA#123]
I've made some changes and refactoring to how Scenery input handling works that generally should NOT cause any changes in behavior. @ariel-phet noted this would be good to get some testing. Changes generally stemmed from https://github.com/phetsims/balloons-and-static-electricity/issues/406, but it included some fixes (to temporarily broken behavior) and general code improvements.

I've done snapshot comparisons before/after indicating it's unlikely I've broken anything dealing with mouse events, so brief spot-checks for mouse and not-quite-as-brief spot-checks for touch (1-2 browsers?) would probably be sufficient.

Basically, it's in master, just run some sims from phettest (pull first) and see if any user input things seem broken.

If it's included, I'm also curious about how fuzzMouse/fuzzTouch now behave. Using `?fuzzTouch` should work out this system (see https://github.com/phetsims/scenery/issues/401), and I'd love to make sure it works on our main supported browsers.

**Comment by phet-steele** [comment]
@lmulhall-phet can you do some iPads and the Android tablet. That's what I don't have.

**Comment by phet-steele** [comment]
My devices are fine. They are macOS 10.13.4 and Win 10 browsers. Could only use Chrome dev tools to emulate touch.

**Comment by jonathanolson** [comment]
Thanks!
